core/modules/date/templates/date-views-pager.tpl.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @file
+ * Template file for the example display.
+ *
+ * Variables available:
+ * 
+ * $plugin: The pager plugin object. This contains the view.
+ *
+ * $plugin->view
+ *   The view object for this navigation.
+ *
+ * $nav_title
+ *   The formatted title for this view. In the case of block
+ *   views, it will be a link to the full view, otherwise it will
+ *   be the formatted name of the year, month, day, or week.
+ *
+ * $prev_url
+ * $next_url
+ *   Urls for the previous and next calendar pages. The links are
+ *   composed in the template to make it easier to change the text,
+ *   add images, etc.
+ *
+ * $prev_options
+ * $next_options
+ *   Query strings and other options for the links that need to
+ *   be used in the l() function, including rel=nofollow.
+ */
+?>
+<?php if (!empty($pager_prefix)) : ?>
+<?php print $pager_prefix; ?>
+<?php endif; ?>
+<div class="date-nav-wrapper clearfix<?php if (!empty($extra_classes)): print $extra_classes; endif; ?>">
```

Comment:
These classes should be an array in preprocess, and then printed here using `<?php print implode(' ', $classes); ?>` to match with how we handle classes in Backdrop template files.